lldb/include/lldb/Symbol/TypeSystem.h

```diff
@@ -384,13 +384,15 @@ class TypeSystem : public PluginInterface,
   dump(lldb::opaque_compiler_type_t type) const = 0;
 #endif
 
-  virtual void DumpValue(lldb::opaque_compiler_type_t type,
-                         ExecutionContext *exe_ctx, Stream &s,
-                         lldb::Format format, const DataExtractor &data,
-                         lldb::offset_t data_offset, size_t data_byte_size,
-                         uint32_t bitfield_bit_size,
-                         uint32_t bitfield_bit_offset, bool show_types,
-                         bool show_summary, bool verbose, uint32_t depth) = 0;
+  /*
+    virtual void DumpValue(lldb::opaque_compiler_type_t type,
+                           ExecutionContext *exe_ctx, Stream &s,
+                           lldb::Format format, const DataExtractor &data,
+                           lldb::offset_t data_offset, size_t data_byte_size,
+                           uint32_t bitfield_bit_size,
+                           uint32_t bitfield_bit_offset, bool show_types,
+                           bool show_summary, bool verbose, uint32_t depth) = 0;
+                           */
 
```

Comment:
remove this, don't just comment it out?

Reply:
I had updated the patch where I effectively removed it. Sorry, it seems you saw the oldest version.

Reply:
It is still here even with "show all commits"?

Reply:
lol, I had stashed the change where I delete the code. 
@clayborg , now this patch should be fine